Lib/ensurepip/__init__.py

```diff
@@ -1,8 +1,10 @@
 import os
 import os.path
-import pkgutil
 import sys
 import tempfile
+from . import _bundled
+
+from importlib.resources import read_binary
```

Comment:
Import to the module, not the object (both to follow the style of the module and because it avoids circular import issues):
```suggestion
from importlib import resources
```

Reply:
IIRC, I was not able to do the import without adding the __init__.py to make it  a package.